app/controllers/movies_controller.rb

```diff
@@ -1,13 +1,17 @@
 class MoviesController < ApplicationController
   def index
-    if params[:search]
-
+    if params[:title]
+      response1 = Faraday.get "https://api.themoviedb.org/3/search/movie?api_key=#{ENV['TMDB_API_KEY']}&language=en-US&query=#{params[:title]}&page=1&include_adult=false"
+      response2 = Faraday.get "https://api.themoviedb.org/3/search/movie?api_key=#{ENV['TMDB_API_KEY']}&language=en-US&query=#{params[:title]}&page=2&include_adult=false"
+      body1 = JSON.parse(response1.body, symbolize_names: true)
+      body2 = JSON.parse(response2.body, symbolize_names: true)
+      @movies = (body1[:results] << body2[:results]).flatten
     else
-      response_1 = Faraday.get "https://api.themoviedb.org/3/movie/top_rated?api_key=#{ENV['TMDB_API_KEY']}&page=1"
-      response_2 = Faraday.get "https://api.themoviedb.org/3/movie/top_rated?api_key=#{ENV['TMDB_API_KEY']}&page=2"
-      body_1 = JSON.parse(response_1.body, symbolize_names: true)
-      body_2 = JSON.parse(response_2.body, symbolize_names: true)
-      @top_40 = (body_1[:results] << body_2[:results]).flatten
+      response1 = Faraday.get "https://api.themoviedb.org/3/movie/top_rated?api_key=#{ENV['TMDB_API_KEY']}&page=1"
+      response2 = Faraday.get "https://api.themoviedb.org/3/movie/top_rated?api_key=#{ENV['TMDB_API_KEY']}&page=2"
+      body1 = JSON.parse(response1.body, symbolize_names: true)
+      body2 = JSON.parse(response2.body, symbolize_names: true)
+      @movies = (body1[:results] << body2[:results]).flatten
     end
   end
 end
```

Comment:
RuboCop threw concerns for method and string lengths being too long. This controller will be refactored soon.